src/components/OAuthLoginButton.tsx

```diff
@@ -1,97 +1,55 @@
-import React, { FC, useCallback } from 'react';
-import {
-  Platform,
-  TouchableOpacity,
-  TouchableOpacityProps,
-  View,
-  ViewStyle,
-  Text,
-  TextStyle,
-} from 'react-native';
+import React, { useCallback } from 'react';
+import { TextStyle, TouchableOpacityProps, ViewStyle } from 'react-native';
 import { useStyles } from '../../src/hooks/useStyles';
 import { LoginParams, useOAuthFlow } from '../../src/hooks/useOAuthFlow';
-import { tID } from '../common/testID';
 import { Theme } from './BrandConfigProvider';
+import { Button } from 'react-native-paper';
 
-type OAuthLoginButtonParams = Omit<TouchableOpacityProps, 'onPress'> &
+type Props = Omit<TouchableOpacityProps, 'onPress'> &
   LoginParams & {
     label: string;
     styles?: OAuthLoginButtonStyles;
   };
 
-export const OAuthLoginButton: FC<OAuthLoginButtonParams> = ({
+export function OAuthLoginButton({
```

Comment:
Just curious, what was your motivation for converting this from an arrow function?

Reply:
Thanks for the detailed explanation, I like it and agree. We should see if we can get a lint rule in place to ensure we follow this pattern moving forward (provided we all agree).